src/server/response.cpp

```diff
@@ -200,6 +201,7 @@ void ContentResponse::introduce_taskbar()
   data.set("root", m_root);
   data.set("content", m_bookName);
   data.set("hascontent", !m_bookName.empty());
+  data.set("hasTitle", !m_bookTitle.empty());
   data.set("title", m_bookTitle);
```

Comment:
Maybe we can regroup the bookName and bookTitle under the same mustache structure (`book` or `bookInfo`) and then simply test if this object is empty in the template.

Reply:
@mgautierfr I did not do this since the taskbar renders when `bookName` is correct but does not have a valid `bookTitle`. There needed to be a separate check for `bookTitle`.

Reply:
I'm bit disturbed by this.

The bookName is the identifier of a book. If it is valid (we can get a reader for it), we have a bookTitle. If it is invalid, then we should not use it in the taskbar (at least, not as the `content` variable, we may still the value somewhere else (error message, details,...))

If the template is buggy, we should fix it. Not add buggy api around it.